### ISSUE TITLE
apigateway - support AuthorizationScopes property on Method resource

### ISSUE THREAD

**Issue by fogfish** [aws/aws-cdk#6390]
Usage of Cognito with API GW requires definition of OAuth scopes otherwise all API requests get rejected. 

Here is snippet of method definition:

```ts
const proxy = rest.root.addResource('{any+}')
proxy.addMethod("ANY", x.lambda, {
   authorizer: {authorizerId: x.auth.ref},
   authorizationType: api.AuthorizationType.COGNITO,
   // authorizationScopes <-- absolutely needed here
   requestParameters: {
        "method.request.header.Authorization": true,
   }
})
```

`CfnMethod` supports the property
https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-apigateway.CfnMethod.html

but `MethodOptions` does not support it
https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-apigateway.MethodOptions.html

and  `Method` class does not pass 'em to `CfnMethod`
https://github.com/aws/aws-cdk/blob/v1.25.0/packages/@aws-cdk/aws-apigateway/lib/method.ts#L150

### Environment

  - **CLI Version      :** 1.25.0
  - **Framework Version:**
  - **OS               :** MacOS
  - **Language         :** typescript

--- 

This is :bug: Bug Report

**Comment by Pibotasnus** [comment]
I have the same problem, I can't add my scopes to my endpoints

**Comment by shuvabrata** [comment]
Yes, would love to have this feature. Right now, we have to document to do this step manually after every CDK deployment.

**Comment by nija-at** [comment]
Re-classified this as a feature request. Missing properties are not bugs in the CDK.

@fogfish  & @shuvabrata  - Until this is available, you can always use the [cfn layer](https://docs.aws.amazon.com/cdk/latest/guide/cfn_layer.html) to workaround this and add it as part of your CDK app.

**Comment by fogfish** [comment]
Thank you!

Do you mean the chapter:
Modifying the AWS CloudFormation Resource behind AWS Constructs

**Comment by nija-at** [comment]
Yes, that's correct.

**Comment by pankajy93** [comment]
@fogfish, @shuvabrata , @Pibotasnus  use cdk version 1.28.0 to configure authorizationScopes.

**Comment by nija-at** [comment]
This has been implemented in 1.28.0.
